content/2019/06/17/refactrtech.rst

```diff
@@ -0,0 +1,168 @@
+.. _This Dot Labs: https://www.thisdot.co/labs
+.. _Dr. Kortney Ziegler: https://twitter.com/fakerapper?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor
+.. _ZaMLabs: https://sites.google.com/zamlabs.info/medialab/about
+
+Track sessions included Front-End Engineering, Software Engineering, Product &
+UX, Cool Sh!t, Career & Leadership, and Social Impact. I tried my best to get in
+on talks from each track. Speakers with various pathways into technology and
+levels of experience brought their own voices to present their passions, ideas,
+and tips and tricks.
+
+`Mike Harington`_ from the `Ionic`_ team went beyond the basics of what makes a
+website a progressive web app and demonstrated through code and personal
+experience which of the big three JavaScript frameworks offers the most support
+out of the box.
+
+.. _Mike Harington: https://twitter.com/mhartington
+.. _Ionic: https://ionicframework.com/
+
+`Tae’lur Alexis`_ of `codeeveryday.io`_ gave us the tea on her newfound passion
+with accessibility. The inspiration for her talk just might have been divine
+providence as she’s been apparently working through issues with her eyesight.
+
+.. _Tae’lur Alexis: https://twitter.com/TaelurAlexis
+.. _codeeveryday.io: https://codeeveryday.io/
+
+I got to sit in on perhaps `the most diverse panel of technologists`_ I’ve ever
+seen giving clutch information on what it’s like betting on yourself through
+entrepreneurship, building your community, and funding your ideas in a world
+where venture-capitalists may overlook you for being a woman or a minority.
+
+.. _the most diverse panel of technologists: https://refactr.tech/detail/sessions.html#forging-a-path-through-inclusive-entrepreneurship
+
+`Sangeeta Nori`_ from GM demystified `quantum computing`_ for us and
+demonstrated how companies are leveraging this very complicated technology to
+solve problems of today and tomorrow. My immediate thought was: *How can this be
+used to tackle climate change, perhaps the most pressing issue of our time?* It
+was fun to chat her up while playing Uno during happy hour, to learn that she is
+struggling to learn Angular during her day job, just like me.
+
+.. _Sangeeta Nori: https://twitter.com/norisangeeta
+.. _quantum computing: https://en.wikipedia.org/wiki/Quantum_computing
+
+I sat in on a panel with A* members of `/dev/color`_ from Netflix on being
+authentic leading teams while black in Silicon Valley  –  showing there is
+strength in a shared experience and knowing that you’re not alone.
+
+.. _/dev/color: https://www.devcolor.org/
+
+`Alexandra Millatmal`_ of `Newsela`_ channeled her personal “othered” experience
+in her talk, "Mentoring the way to diversity and inclusion". It was a
+call-to-action for companies to make cultural shifts in how they hire, onboard
+and mold upcoming talent; talent that perhaps have less-than-conventional paths
+to get where they are – and how mentoring junior engineers and bolstering D&I
```

Comment:
I'd love to hear more about this, personally. Maybe I just need to go watch the talk. :)

Reply:
Ditto here on viewing the talks.  This speaker had good energy 🧘🏽.